BookingProject/BookingWebAPI/AuditWebApi.Infrastructure/Kafka/KafkaConsumer.cs

```diff
@@ -0,0 +1,86 @@
+﻿using AuditWebApi.Application;
+using AuditWebApi.Application.DTOs;
+using AuditWebApi.Infrastructure.Configuration;
+using Confluent.Kafka;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace AuditWebApi.Infrastructure.Kafka
+{
+    public class KafkaConsumer : BackgroundService
+    {
+        private readonly ILogger<KafkaConsumer> _logger;
+        private readonly IConsumer<Ignore, string> _consumer;
+        private readonly string _topic;
+        //private readonly IAuditService _auditService;
+        private readonly IServiceScopeFactory _scopeFactory;
+        public KafkaConsumer(IOptions<KafkaSettings> options, ILogger<KafkaConsumer> logger, IServiceScopeFactory scopeFactory)
+        {
+            try
+            {
+                var config = new ConsumerConfig
+                {
+                    BootstrapServers = options.Value.BootstrapServers,
+                    GroupId = "audit",
```

Comment:
Move consumer group to appsettings. Each application (monolith, audit, finance service, etc) should have it's own consumer group.
This way we:
1. Are sure that each application can consume events independently from each other
2. We can horizontally scale the applications. Since all instances of audit service will have "audit" consumer group, this means that they will be able to consume from different partitions without conflicts